images/archive/exporter.go

```diff
@@ -170,8 +170,7 @@ func Export(ctx context.Context, store content.Provider, writer io.Writer, opts
 	dManifests := map[digest.Digest]*exportManifest{}
 	resolvedIndex := map[digest.Digest]digest.Digest{}
 	for _, desc := range eo.manifests {
-		switch desc.MediaType {
-		case images.MediaTypeDockerSchema2Manifest, ocispec.MediaTypeImageManifest:
+		if images.IsManifestType(desc.MediaType) {
```

Comment:
Alternatively, you could keep a switch, and use the boolean checks in it (I personally find those to be slightly more readable as the condition is more "in plain sight"), but that's really a bike-shed, so just leaving the comment for consideration 😄 

```go
switch {
case images.IsManifestType(desc.MediaType):
    // ...
case images.IsIndexType(desc.MediaType):
    // ...
default:
    // ...
}
```

Reply:
I tend to prefer the if and braces style for relatively dense blocks like this; unless another review prefers this, I think I'd like to leave this how I have it.